index.js

```diff
@@ -65,12 +74,16 @@ module.exports = {
     let bowerDependencies = this.app.project.bowerDependencies();
     switch (name) {
       case 'sass':
-        if (!('bootstrap-sass' in npmDependencies)) {
+        if (!('bootstrap-sass' in npmDependencies) && this.getBootstrapVersion() === 3) {
           this.ui.writeLine(chalk.red('Npm package "bootstrap-sass" is missing, but required for SASS support. Please run `ember generate ember-bootstrap` to install the missing dependencies!'));
           return false;
         }
         break;
       case 'less':
+        if (this.getBootstrapVersion() === 4) {
+          this.ui.writeLine(chalk.red('There is no Less support for Bootstrap 4! Falling back to importing static CSS. Consider switching to Sass for preprocessor support!'));
+          return false;
+        }
         if (!('bootstrap' in bowerDependencies)) {
```

Comment:
I was going to say it wasn't checking the bootstrap version, but hadn't fully thought through the effect of the conditional above it.